data/data/appeals-service-api/001-appeals.json

```diff
@@ -1 +1,99 @@
-[]
+[
```

Comment:
Is this file part of the appeals data panel? Should it be part of a wider story on setting up data?

Reply:
This is the first instance of using the data so needs the setup. Same as the other stories - the setup has been made part of the story

Reply:
We can always add more pieces of seed data as needed - I would be surprised if this were the only piece of seed data we ever need